pkg/systemd/overview-cards/systemInformationCard.jsx

```diff
@@ -126,7 +126,7 @@ export class SystemInformationCard extends React.Component {
                                 </td>
                             </tr>
                             <tr className="pf-v5-c-table__tr">
-                                <th className="pf-v5-c-table__th system-information-uptime" scope="row">{_("Uptime")}</th>
+                                <th className="pf-v5-c-table__th system-information-uptime" scope="row">{_("Started")}</th>
```

Comment:
I liked @allisonkarlitskaya suggestion of `Up since`

Reply:
I like it too -- changed to that. (Took a while due to the pixel tests)